app/assets/stylesheets/_search_result.scss

```diff
@@ -37,22 +38,28 @@
           margin: 10px auto;
           img { margin: 0; }
         }
-      }
+       }
+     }
+    .button-action {
+      float: left;
+      padding-top: 10px;
     }
-    .examples-container {
-      .translation-gloss {
-        @include montserrat-regular;
-        font-size: $h6-font-size;
-        margin: 0 0 10px 0;
-      }
+    .add-to-vocab-btn { padding-right: 45px; }
+  }
+
+  .examples-container {
+    .translation-gloss {
+      @include montserrat-regular;
+      font-size: $h6-font-size;
+      margin: 0 0 10px 0;
     }
   }
 }
 
 @media screen and (min-width: $medium-screen) {
   .search-result-container {
     .sign-container { float: right; }
-    .add-to-vocab-btn,
+    .add-to-vocab-btn { float: right; }
```

Comment:
Should the positioning for this file change to flex? Is that on the horizon with another branch?

Reply:
+1 to this, Floats can be pretty finicky. Big thumbs up for using flex instead if we can

Reply:
@Dan @danieljauch I have added an existing class with a position of flex. Putting flex on this `.add-to-vocab-btn` resulted in weird positioning.